src/actions/resetPasswordActions.js

```diff
@@ -0,0 +1,47 @@
+import types from './actionTypes';
+import store from '../store';
+import axiosInstance from '../services/requestHandler';
+
+const { dispatch } = store;
+
+const { RESET_PASSWORD_LINK, RESET_PASSWORD,
+  RESET_PASSWORD_TOKEN } = types;
+
+const resetPassword = (path, password) => (
+  {
+    type: RESET_PASSWORD,
+    payload: axiosInstance().put(
+      path,
+      { user: { password, confirm: password } },
+    ),
+  }
+);
+
+const resetPasswordLink = (path, email) => (
+  {
+    type: RESET_PASSWORD_LINK,
+    payload: axiosInstance().post(path, { user: { email } }),
+  }
+);
+
+
+export const setToken = token => (
+  {
+    type: RESET_PASSWORD_TOKEN,
+    token,
+  }
+);
+
+const sendResetPasswordLink = (path, email) => {
+  dispatch(resetPasswordLink(path, email));
```

Comment:
I had previously said not to use dispatch here since it seems sorta redundant to still call that sae action in side the same file

Reply:
I have made the adjustment